code/datums/map_config.dm

```diff
@@ -40,7 +40,8 @@
 	var/job_changes = list()
 	/// List of additional areas that count as a part of the library
 	var/library_areas = list()
-
+	/// What day/night controller should we add when this map is loaded? if any.
+	var/day_night_controller
```

Comment:
If this is a path, ought to be suffixed with _path.

Reply:
ya